tests/test_nc_to_mmd.py

```diff
@@ -527,5 +527,15 @@ def test_create_mmd_missing_abstract(self):
         self.assertEqual(md.missing_attributes['errors'][0],
                 'summary is a required ACDD attribute')
 
+    def test_publication_date(self):
+        format = '%Y-%m-%d'
+        mmd_yaml = yaml.load(resource_string('py_mmd_tools', 'mmd_elements.yaml'), Loader=yaml.FullLoader)
+        nc2mmd = Nc_to_mmd('tests/data/reference_nc.nc')
+        ncin = Dataset(nc2mmd.netcdf_product)
+        value = nc2mmd.get_dataset_citations(mmd_yaml['dataset_citation'], ncin)
+        dt = datetime.datetime.strptime(value[0]['publication_date'], format)
+        self.assertEqual(dt, datetime.datetime(2020, 11, 27, 0, 0))
+
```

Comment:
Why are you going via datetime?

so value[0]['publication_date'] is a string?

So why not `self.assertEqual(value[0]['publication_date'], '2020-11-27')` or something like that?

Reply:
to be sure it has the right format... but strictly speaking you're right - it's not really needed. I suggest to leave it as it is